rudra/deployments/emr_scripts/emr_config.py

```diff
@@ -62,6 +63,12 @@ def get_config(self):
 
         execute_training_code = ['python3.6'.format(self.repo_dir),
                                  training_file, self.hyper_params]
+
+        if self.ecosystem=='npm':
+            training_file = "{}/{}".format(self.repo_dir, self.training_file_name)
```

Comment:
training_file is already set earlier. No need to do it again

Reply:
Ok!!